Library/Formula/swiftgen.rb

```diff
@@ -0,0 +1,19 @@
+class Swiftgen < Formula
+  desc "A collection of Swift tools to generate Swift code (enums for your assets, storyboards, Localizable.strings, …)"
+  homepage "https://github.com/AliSoftware/SwiftGen"
+
+  url "https://github.com/AliSoftware/SwiftGen/archive/0.4.1.tar.gz"
+  sha256 "e572631bdac5ebff49314f56cd4bd25a0eae8efc2390602e30d6cac1f196f53b"
+  head "https://github.com/AliSoftware/SwiftGen.git", :branch => "master"
+
+  def install
+    system "rake", "all[#{prefix},/bin]"
+  end
+
+  test do
+    system "#{bin}/swiftgen-assets", "--version"
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!